src/AlternateLife.RageMP.Net/Scripting/ScriptingClasses/TypeParsers/EnumParser.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using AlternateLife.RageMP.Net.Interfaces;
+
+namespace AlternateLife.RageMP.Net.Scripting.ScriptingClasses.TypeParsers
+{
+    internal class EnumParser : ITypeParser
+    {
+        public bool TryParse(string s, Type targetType, out object value)
+        {
+            value = default(object);
```

Comment:
This step is not needed, because `Enum.TryParse` will already fill `value`, so you could reduce the size of this method to
```cs
return Enum.TryParse(targetType, s, true, out value);
```